docs/tutorials/user/credit-class-project-batch-management.md

```diff
@@ -0,0 +1,468 @@
+# Credit Class, Project, and Batch Management
+
+This tutorial covers the creation and management of credit classes, projects, and batches using the [command-line interface (CLI)](../../ledger/infrastructure/interfaces.html#command-line-interface). This tutorial will demonstrate with data standards and practices used by Regen Network Development for [Regen Registry Program](https://library.regen.network/).
+
+For information about creating and managing credit classes, projects, and credit batches using the [Regen Marketplace](https://app.regen.network/) application, see [Regen Network Guidebook](https://guides.regen.network/guides/regen-marketplace).
+
+## Prerequisites
+
+- [Install Regen](../../ledger/get-started)
+- [Redwood Testnet](../../ledger/get-started/live-networks.md#redwood-testnet)
```

Comment:
It might be nice to explicitly tell people how to use the `$REGEN_HOME` environment variable to setup a testnet home directory that they can easily swap between.

A few lines after the prerequisites that is just "setting up your redwood config":
```
export REGEN_HOME=~/.regen-redwood
regen config chain-id regen-redwood-1
regen config node http://redwood.regen.network:26657
```

And how they can then use that `~/.regen-redwood` home dir whenever (by exporting the env var).

Reply:
I think we can update our getting started documentation in a separate pull request. :+1:

Reply:
I was thinking we break out Regen Mainnet and Redwood Testnet into pages and make improvements there. I didn't make any changes to the tutorial in this pull request though.